src/components/CorporateCampaign/CampaignLoanRow.vue

```diff
@@ -203,12 +209,16 @@ export default {
 		}
 	},
 	methods: {
+		// TODO: Review all tracking cateogries
+		onInteractCarousel(interaction) {
+			this.$kvTrackEvent('carousel', 'click-carousel-horizontal-scroll', interaction);
+		},
 		addToBasket(payload) {
 			this.loanAdded = true;
 			this.$emit('add-to-basket', payload);
 		},
 		showLoanDetails(payload) {
-			const selectedLoan = this.loans.find(loan => loan.id === payload.loanId);
+			const selectedLoan = this.loans.find(loan => String(loan.id) === String(payload.loanId));
 			this.$emit('show-loan-details', selectedLoan);
```

Comment:
Does this method need to `preventDefault` or anything like that to ensure the clicks coming up from the child components do not navigate?